### PR TITLE
add fonttong.rb

### PR DESCRIPTION
**Pull request by daehyunpy**
I added fonttong.rb for fonttong app

### CODE REVIEW

File: Casks/fonttong.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'fonttong' do
+  version '2.0'
+  sha256 '70974dcdc39bb6f49a3440a265c3f6f87df0d5cf2b843e22fbd3120b9ce428a1'
+
+  url 'http://www.asiafont.com/FontTong.dmg'
+  name 'FontTong'
```
Comment (vitorgalvao) [inline Casks/fonttong.rb]: The original name should also be added.

File: Casks/fonttong.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  homepage 'http://www.asiafont.com'
+  license :gratis
+
+  app "폰트통v#{version}.app", :target => 'FontTong.app'
```
Comment (vitorgalvao) [inline Casks/fonttong.rb]: Why are you renaming it? This doesn’t seem to be [a case for it](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#target-should-only-be-used-in-select-cases).

**Comment by hyuna917** [comment]
I added original name. and I renamed it and now it is easy to run with Spotlight and shell.

**Comment by vitorgalvao** [comment]
That’s not an acceptable case, though. The developer named the app in such a manner for a reason, and there’s no reason to change it. You’re making it easier for you, but perhaps harder for their target audience. Especially whith #13201, where this would rename the app itself, this makes no sense.

Please remove `:target`. If you need another name, you’ll need to do it in your setup, or ask the developer.

**Comment by hyuna917** [comment]
I change the name to original.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**